Create 2025-09-10-TAC.md (#100)
PQCA/TAC: meeting-minutes/2025/2025-09-10-TAC.md

**LF Staff**

* **Christina Harter**  
* **Hart Montgomery**   
* **Min Yu** 

**Other Attendees**

* **Aditya Koranga, NgKore \[TAC Vice Chair\]**  
* **Miguel Diaz Goncalves**

---

### **Meeting Agenda**

Brian Jarvis presented the meeting agenda:

* Welcome and introduction of any new attendees  
* Follow-ups:  
  * Project lifecycle stage for OQS ([Discussion](https://github.com/orgs/open-quantum-safe/discussions/2191))  
  * PQCA wiki added to website ([wiki.pqca.org](http://wiki.pqca.org)); likely close [Issue \#91](https://github.com/PQCA/TAC/issues/91).  
  * Tooling WG project transition pending IBM legal review of charter.  
  * Lifecycle document annual review: [Issue \#93](https://github.com/PQCA/TAC/issues/93) | [Discussion \#95](https://github.com/PQCA/TAC/discussions/95%20)  
* Project status updates:  
  * OQS  
  * PQCP  
  * Tooling WG

---

### **Introduction of 1st Time Attendees**

Min Yu was welcomed as the new PQCA Program Manager (along with Christina Harter). She expressed enthusiasm for supporting the community and assisting with program management efforts.

---

### **Project Metrics and Lifecycle Updates**

> * OQS and PQCP lifecycle stages finalized via TAC vote.  

Reviewer: ```suggestion
* OQS lifecycle stage finalized via TSC vote, selecting Research--Labs stage
* PQCP lifecycle stage was finalized previously, selecting Production--Growth stage
```

* Aditya to update lifecycle stages on the GitHub wiki and explore LFX Insights integration.  

> * Hart offered to share resources and templates from other LF projects to define PQCP growth metrics; to be discussed during the October PQCP meeting.

Reviewer: ```suggestion
* Hart offered to share resources and templates from other LF projects to define PQCP growth plan; to be discussed during the October PQCP meeting.
```

---

### **Project Updates and Process Changes**

* PQCA wiki subdomain ([wiki.pqca.org](http://wiki.pqca.org)) now live & added to PQCA website; issue \#91 to be closed.  

> * Andreas to follow up on CBOM kit project transition post-vacation.  

Reviewer: Minor nit on how CBOMkit is stylized:
```suggestion
* Andreas to follow up on CBOMkit project transition post-vacation.  
```

* Norman created a PR to update the lifecycle document based on JP’s comments; feedback pending.  
* Brian suggested running another retrospective with Max to review PQCA processes and improvements.

---

### **Project Status Updates**

* **OQS:** Lifecycle classification finalized; next steps include documenting metrics and contribution pathways.  

> * **PQCP:** Planning upcoming growth strategy; Hart to collaborate on project health indicators.  

Reviewer: ```suggestion
* **PQCP:**
  * mlkem-native: received a contribution from IBM for a PowerPC backend. Verification of this PR is underway.
  * mldsa-native: working on closing the performance gaps relative to the state of the art
  * Beginning to work with a new mentee
  * Preparing an upcoming release of libjade which includes additional ML-KEM parameter sets
  * Planning upcoming growth strategy; Hart to collaborate on project health indicators.  
```

> * **Tooling WG:** CBOM kit proof-of-concept underway at Siemens Healthineers; ongoing charter transition discussions continue.

Reviewer: ```suggestion
* **Tooling WG:** CBOMkit proof-of-concept underway at Siemens Healthineers; ongoing charter transition discussions continue.
```

---

### **Open Action Items / Issues Review**

| Action Item | Owner | Status / Due Date |
| ----- | ----- | ----- |
| Close PQCA wiki discoverability issue (\#91) | Brian | Done |
| Update GitHub wiki with OQS & PQCP lifecycle stages | Aditya | Before next meeting |
| Organize retrospective with Max | Brian | Before next meeting  |

> | Follow up on CBOM kit project transition | Andreas | Next meeting |

Reviewer: ```suggestion
| Follow up on CBOMkit project transition | Andreas | Next meeting |
```
